### DISCUSSION TITLE
Using OpenXR

### DISCUSSION THREAD

**Discussion by courgeon** [dotnet/Silk.NET#608] [Q&A]
Hello, I'm trying to use OpenXR bindings, (with a vive cosmos, SteamVR being setup as the current OpenXR environment)

but just doing 
                `var Xr = XR.GetApi();`
result in 
                `System.IO.FileNotFoundException : 'Could not find or load the native library: openxr_loader.dll Attempted: "openxr_loader.dll"`

What do I need to install ? What I am missing ?

M.

**Comment by HurricanKai** [comment]
You need some kind of OpenXR runtime. See the OpenXR landing page: https://www.khronos.org/openxr/ it lists a couple

**Reply by Perksey** [reply]
You also need to build the OpenXR loader and put the binary next to your application.
